examples/utils.py

```diff
@@ -0,0 +1,12 @@
+def render_a_trajectory_and_print_reward(env, policy):
```

Comment:
My gut feeling says, that this function should already be in some gym utils, but I could not find it. Does anybody have a hint for me?

Reply:
You could do this using `InteractiveRender` from https://github.com/HumanCompatibleAI/imitation/blob/master/src/imitation/scripts/eval_policy.py#L22 (move it from script to util), `rollout.generate_trajectories` and `rollout.rollout_stats`. Would need to wrap the env into a `VecEnv` first. Not obviously cleaner given the number of utilities needed, but introduces the reader to some relevant code at least.

Reply:
I went with `stable_baselines3.common.evaluation.evaluate_policy` for now. There is a `render` flag, that we can just set to `True`.